platform-sdk/swirlds-common/build.gradle.kts

```diff
@@ -15,37 +15,38 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("org.gradle.java-test-fixtures")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
+    id("com.swirlds.platform.maven-publish")
+    id("org.gradle.java-test-fixtures")
 }
 
 dependencies {
-  // Individual Dependencies
-  api(project(":swirlds-base"))
-  api(project(":swirlds-logging"))
-  api(project(":swirlds-config-api"))
-  implementation(project(":swirlds-base"))
-  implementation(libs.classgraph)
-  implementation(libs.commons.codec)
-  implementation(libs.prometheus.httpserver) {
-    exclude("io.prometheus", "simpleclient_tracer_otel")
-    exclude("io.prometheus", "simpleclient_tracer_otel_agent")
-  }
-  compileOnly(libs.spotbugs.annotations)
+    // Individual Dependencies
+    api(project(":swirlds-base"))
+    api(project(":swirlds-logging"))
+    api(project(":swirlds-config-api"))
```

Comment:
Are the formatting changes intentional?

Reply:
Looks like spotless did it since it was changed for the first time?